src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -163,10 +163,11 @@ public function update_item( $request ) {
 			return $fields_update;
 		}
 
-		return $this->prepare_item_for_response(
-			get_post( $request['id'] ),
-			$request
-		);
+		wp_after_insert_post( $post, true, $post_before );
+
+		$response = $this->prepare_item_for_response( $post, $request );
```

Comment:
Before this PR, $post was re-retrieved with `get_post( $request['id'] )`, presumably to account for anything changed as a result of `update_additional_fields_for_object`. After this PR, it is not re-retrieved anymore. Is this intentional, or is it a bug?

Reply:
This is how it works on the post controller. Just copying that format over. 

https://github.com/WordPress/wordpress-develop/blob/7b9e33a1e76c5052b57f7a701e5ec228b3751dea/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L884-L905